101-azurefirewall-with-firewallpolicy-sandbox-linux/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "environments": [
+    "AzureCloud"
+  ],
+  "itemDisplayName": "Create a sandbox setup of Azure Firewall using Firewall Policy with Linux VMs",
```

Comment:
display name is too long - open the file in VS code to check schema issues

Reply:
Fixed